### ISSUE TITLE
Clarify that separators, resize handlers, scrollbars are also covered in 2.5.8 target size

### ISSUE THREAD

**Issue by gundulaniemann** [w3c/wcag#3629]
In order to use all functionality that is required to complete a process, a user might need to

- resize an object, for example a dialog, to fully see the content
- move a separator, for example to fully see a panel or header of a page
- scroll the page or a part of the page
When using a pointer device, the corresponding handlers are targets for pointer input

As we received numerous feedback asking whether target size also applies for such handlers, it does not seem to be clear these pointer operations are also covered by 2.5.8.
Can you please add an according remark?

**Comment by patrickhlauke** [comment]
is this a duplicate of https://github.com/w3c/wcag/issues/3628 ?

**Comment by gundulaniemann** [comment]
indeed, the functionalities in question are the same for different Success Criteria where I ask

**Comment by detlevhfischer** [comment]
My understanding would be that pointer actions implemented by authors' scripting would fall under 2.5.8 (say, a title bar of a custom dialog that can be dragged elsewhere by picking it up at that bar), but functionality provided by the underlying technology and implemented by the user agent (e.g. the CSS `resize` attribute that creates the triangular handle in the corner of a `textarea` or `div`) would not need to meet 2.5.8 because the author is not responsable for the implementation (and may not even be able to modify it).

**Comment by patrickhlauke** [comment]
I believe this has been addressed by https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum#user-agent-control
